src/bio/smd/smd_store.c

```diff
@@ -270,6 +271,12 @@ smd_store_open(char *fname)
 	if (smd_df->smd_version > SMD_DF_VERSION ||
 	    smd_df->smd_version < SMD_DF_VER_1) {
 		D_ERROR("Unsupported DF version %d\n", smd_df->smd_version);
+		if (ds_notify_df_incompat != NULL) {
+			/** Send a RAS notification */
+			ds_notify_df_incompat("SMD pool", smd_df->smd_version,
```

Comment:
I tested the VOS part but haven't yet tested this case as I had issues with NVMe configuration